components/expenses/PayoutBankInformationForm.js

```diff
@@ -106,24 +106,29 @@ const RequiredFields = ({ disabled, getFieldName, formik, collective, currency }
         input.key === 'accountHolderName'
           ? getFieldName(`data.${input.key}`)
           : getFieldName(`data.details.${input.key}`);
-
+      let validate;
       switch (input.type) {
         case 'text':
+          validate = input.validationRegexp
+            ? value => (new RegExp(input.validationRegexp).test(value) ? undefined : `Invalid ${input.name}`)
+            : undefined;
           return (
-            <FastField key={input.key} name={fieldName}>
-              {({ field }) => (
-                <StyledInputField label={input.name}>
+            <FastField key={input.key} name={fieldName} validate={validate}>
+              {({ field, meta }) => (
+                <StyledInputField label={input.name} required error={meta.touched && meta.error}>
                   {() => <Field as={StyledInput} placeholder={input.example} {...field} disabled={disabled} />}
                 </StyledInputField>
               )}
             </FastField>
           );
         case 'radio':
         case 'select':
+          validate = value =>
+            input.valuesAllowed.some(v => v.key === value) ? undefined : 'This value is not accepted.';
           return (
-            <FastField key={input.key} name={fieldName}>
-              {({ field }) => (
-                <StyledInputField label={input.name}>
+            <FastField key={input.key} name={fieldName} validate={validate}>
+              {({ field, meta }) => (
+                <StyledInputField label={input.name} required error={meta.touched && meta.error}>
```

Comment:
Why not using a `StyledSelect` to make sure only valid options can be picked?

Reply:
I'm not sure if I get it, the input child is a `StyledSelect` (line 135).

Reply:
Indeed I missed that. Is the check above (`input.valuesAllowed.some(v => v.key === value) ? undefined : 'This value is not accepted.';`) still necessary?

Reply:
You're right, it is an overkill since I could just check if the value exists.